config.js

```diff
@@ -44,8 +44,9 @@ const config = {
       '/xr_design_and_accessibility',
     ],
     collapsedNav: ['/tutorials'],
-    links: [{ text: 'Pluto', link: 'https://pluto.app' }],
-    frontline: false,
+    collapsedNav: ['/platforms'],
```

Comment:
Probably want these in a single array off of that key, yeah? (Looks like we went from collapsing one to collapsing the other.)

Reply:
oops! Absolutely, thanks.